### PR TITLE
Use external get-func-name module

### PR DESCRIPTION
**Pull request by lucasfcosta**
So, this includes a little "refactor" for us to use `get-func-name` instead of our internal implementation of `getName`.
I also removed redundant tests.

### CODE REVIEW

**Comment by meeber** [comment]
Thanks for doing this. Let's add it to the "Related Projects" section in the README too.

**Comment by lucasfcosta** [comment]
@meeber, great idea, I will update those too then!

**Comment by lucasfcosta** [comment]
@meeber it's done!
Thanks for noticing this.

**Comment by meeber** [comment]
LGTM!

File: lib/chai/utils/index.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ exports.hasProperty = pathval.hasProperty;
  * Function name
  */
 
-exports.getName = require('./getName');
+exports.getName = require('get-func-name');
```
Comment (meeber) [inline lib/chai/utils/index.js]: I don't think it's worth changing.

**Comment by vieiralucas** [comment]
LGTM

+4 -50
Always good to remove code

**Comment by lucasfcosta** [comment]
Thanks for your reviews guys!
Let's get this merged!

🎉 🎉 🎉

**Pull request merged**